### PR TITLE
Major refactor of 6.3 framework release notes

### PR DESCRIPTION
**Pull request by sf1919**
**Description of work.**

The main refactoring the the Framework release notes ahead of release of 6.3. Includes:-

- Removing extraneous headings and developer warning
- Reordering of notes where appropriate
- Rewording notes to improve consistency
- Reformatting of notes.
- Spelling & Grammar check

This PR incorporates some release notes that were saved in separate files (see first commit). These files have been deleted as the release notes have been moved. Further changes will be made later when any additional separate files for v6.3 diffraction release notes are amalgamated into the final release notes.

**To test:**

- remove `build/docs/doctrees`
- build the documents again and check there are no warnings
- Check that the links work
- Check the information is still correct
- Grammar/Spelling check

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.3.0/framework.rst [hunk 1]
```diff
@@ -5,83 +5,73 @@ Framework Changes
...
-- Fix bug in :ref:`Integration <algm-Integration>` when using UsePartialBinsOption with integration limits that are either equal or close together
-- Fix bug in :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` where calculation aborts with exception due to floating point rounding error when track segment close to vertical
-  Also fixed bug in calculation of track direction after scatter if pre-scatter track was pointing exactly down - sign of z component of new direction was incorrect
+- Fixed a bug with :ref:`CalculatePlaczek <algm-CalculatePlaczek>` algorithm for computing Placzek correction factors. There were a few flaws in the previously implemented formulation.
+- Fixed a bug in :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` where calculation aborts with an exception due to a floating point rounding error when the track segment is close to vertical. Also fixed bug in calculation of track direction after scatter if pre-scatter track was pointing exactly down - sign of z component of new direction was incorrect.
```
Comment (DavidFair) [inline docs/source/release/v6.3.0/framework.rst:37]: `where the calculation aborts`

File: docs/source/release/v6.3.0/framework.rst [hunk 2]
```diff
@@ -5,83 +5,73 @@ Framework Changes
...
+- :ref:`SetSampleFromLogs <algm-SetSampleFromLogs>` will now fail if the resulting sample shape has a volume of 0.
 
 Bugfixes
 ########
-
-- Fix bug in :ref:`LoadEventNexus <algm-LoadEventNexus>` in checking valid event ID's and make sure to always exclude data in ``error`` and ``unmapped`` banks.
-- Fix bug in :ref:`Integration <algm-Integration>` when using UsePartialBinsOption with integration limits that are either equal or close together
-- Fix bug in :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` where calculation aborts with exception due to floating point rounding error when track segment close to vertical
-  Also fixed bug in calculation of track direction after scatter if pre-scatter track was pointing exactly down - sign of z component of new direction was incorrect
+- Fixed a bug with :ref:`CalculatePlaczek <algm-CalculatePlaczek>` algorithm for computing Placzek correction factors. There were a few flaws in the previously implemented formulation.
```
Comment (DavidFair) [inline docs/source/release/v6.3.0/framework.rst:36]: Could we find out what those flaws were?
This sort of release note makes users nervous that their data might be wrong

File: docs/source/release/v6.3.0/framework.rst [hunk 3]
```diff
@@ -5,83 +5,73 @@ Framework Changes
...
 - The :ref:`Load <algm-Load>` algorithm now reports the correct history.
-- Fix bug in :ref:`LoadAndMerge <algm-LoadAndMerge>` where LoaderVersion choice was previously ignored
-- Fix bug in :ref:`SaveNexus <algm-SaveNexus>` - ragged workspace x-values are saved correctly when workspace indices are supplied.
-- Fix bug in :ref:`MonteCarloAbsorption <algm-MonteCarloAbsorption>`. If the algorithm was run with the Sparse Workspace feature enabled on a workspace containing spectra
-  that didn't have any detectors it failed with an error
+- Fixed a bug in :ref:`LoadAndMerge <algm-LoadAndMerge>` where ``LoaderVersion`` choice was previously ignored.
+- Fixed a bug in :ref:`LoadEventNexus <algm-LoadEventNexus>` in checking valid event ID's and to make sure to always exclude data in ``error`` and ``unmapped`` banks.
+- Fixed a bug in :ref:`MonteCarloAbsorption <algm-MonteCarloAbsorption>`. If the algorithm was run with the Sparse Workspace feature enabled on a workspace containing spectra
+  that didn't have any detectors, it failed with an error.
+- Fixed a bug in :ref:`SaveNexus <algm-SaveNexus>` - ragged workspace x-values are saved correctly when workspace indices are supplied.
```
Comment (DavidFair) [inline docs/source/release/v6.3.0/framework.rst:44]: are now saved correctly

**Review by DavidFair — CHANGES_REQUESTED** [review]
Some minor typos / corrections
Otherwise reads well

File: docs/source/release/v6.3.0/framework.rst (hunk 2 above)
Comment (sf1919) [reply docs/source/release/v6.3.0/framework.rst:36]: I looked back and this comes from PR #33173 . Looking at the description it seems the transformation constant from k to e was implemented wrong, as was the summation term formulation for second order Placzek corrections. Do we need more information that that? i.e. do we need to explain how they were wrong implemented?

Comment (DavidFair) [reply docs/source/release/v6.3.0/framework.rst:36]: I would keep it at that, `previously implemented formula for transforming k to e, and the summation for second order Placezek corrections`

If people want more details they can reach out or look at the changes without us going into laborious (and possibly wrong) details

**Review by DavidFair — APPROVED** [review]
Builds locally for me without warnings, and the links / inline text all render correctly
Lets :shipit:

**Comment by sf1919** [comment]
@mantidproject/gatekeepers can we get this merged in?

**Pull request merged**